force-app/main/default/classes/RecordManagerTest.cls

```diff
@@ -0,0 +1,58 @@
+@isTest
+private class RecordManagerTest {
+  @isTest
+  static void testGetRecentRecruiterJAAndRelated() {
+    Test.startTest();
+    TestRecordsFactory.insertUser('Recruiter');
+
+    id profileId = [SELECT Id FROM Profile WHERE Name = 'Recruiter' LIMIT 1].Id;
+    User testRecruiter = [SELECT Id FROM User WHERE ProfileId = :profileId AND IsActive = true LIMIT 1];
+    
+    System.runAs(testRecruiter) {
+        TestRecordsFactory.insertReviews(5, '');
+    }
+    
+    Test.stopTest();
+
+    List<Job_Application__c> ja = RecordManager.getRecentRecruiterJAAndRelated();
+    System.assertEquals(
```

Comment:
Use Assert class instead. It's new way for assertion.
https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_class_System_Assert.htm